packages/manager/modules/pci/src/projects/new/new.routing.js

```diff
@@ -0,0 +1,50 @@
+import template from './new.html';
+import controller from './new.controller';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.new', {
+      url: '/new',
+      template,
+      controller,
+      controllerAs: '$ctrl',
```

Comment:
I'm agree but if I do this, it means that I will use a `data-ui-view` inside component template. As we want to avoid the use of `ui-router` features outside of `.routing` files